### DISCUSSION TITLE
Transaction isolation level

### DISCUSSION THREAD

**Discussion by Giorgi** [duckdb/duckdb#4014] [General]
In [.Net library for DuckDB](https://github.com/Giorgi/DuckDB.NET) I have a PR for adding transaction support but we are not sure [what is the isolation level](https://github.com/Giorgi/DuckDB.NET/pull/40#discussion_r911928984) supported by DuckDB.

Could you help us with it?

**Comment by Mytherin** [comment]
DuckDB provides snapshot isolation
